website/docs/guides/container_service_extension_install.html.markdown

```diff
@@ -1,128 +1,167 @@
-user preferences in the top right, then go to the API tokens section, add a new one.
-Or you can visit `/provider/administration/settings/user-preferences` at your VCD URL as CSE Administrator.
+CSE v4.1 requires a set of Runtime Defined Entity items, such as [Interfaces][rde_interface], [Types][rde_type] and [Behaviors][rde_interface_behavior].
+In the [step 1 configuration][step1] you can find the following:
 
-### Step 2: Install CSE
+- The required `VCDKEConfig` [RDE Interface][rde_interface] and [RDE Type][rde_type]. These two resources specify the schema of the **CSE Server
+  configuration** that will be instantiated with a [RDE][rde].
+
+- The required `capvcd` [RDE Interface][rde_interface] and `capvcdCluster` [RDE Type][rde_type].
+  These two resources specify the schema of the [TKGm clusters][tkgm_docs].
+
+- The required [RDE Interface Behaviors][rde_interface_behavior] used to retrieve critical information from the [TKGm clusters][tkgm_docs],
+  for example, the resulting **Kubeconfig**.
+
+#### RDE (CSE Server configuration / VCDKEConfig)
+
+The CSE Server configuration lives in a [Runtime Defined Entity][rde] that uses the `VCDKEConfig` [RDE Type][rde_type].
+To customise it, the [step 1 configuration][step1] asks for the following variables that you can set in `terraform.tfvars`:
+
+- `vcdkeconfig_template_filepath` references a local file that defines the `VCDKEConfig` [RDE][rde] contents.
+  It should be a JSON file with template variables that Terraform can interpret, like
+  [the RDE template file for CSE v4.1](https://github.com/vmware/terraform-provider-vcd/tree/main/examples/container-service-extension/v4.1/entities/vcdkeconfig.json.template)
+  used in the step 1 configuration, that can be rendered correctly with the Terraform built-in function `templatefile`.
+  (Note: In `terraform.tfvars.example` the path for the CSE v4.1 RDE contents is already provided).
+- `capvcd_version`: The version for CAPVCD. The default value is **"1.1.0"** for CSE v4.1.
+  (Note: Do not confuse with the version of the `capvcdCluster` [RDE Type][rde_type],
+  which **must be "1.2.0"** for CSE v4.1 and cannot be changed through a variable).
+- `cpi_version`: The version for CPI (Cloud Provider Interface). The default value is **"1.4.0"** for CSE v4.1.
+- `csi_version`: The version for CSI (Cloud Storage Interface). The default value is **"1.4.0"** for CSE v4.1.
+- `github_personal_access_token`: Create this one [here](https://github.com/settings/tokens),
+  this will avoid installation errors caused by GitHub rate limiting, as the TKGm cluster creation process requires downloading
+  some Kubernetes components from GitHub.
+  The token should have the `public_repo` scope for classic tokens and `Public Repositories` for fine-grained tokens.
+- `bootstrap_vm_sizing_policy`: The [VM Sizing Policy][sizing] used by the Bootstrap VM.
```

Comment:
I think there is no such field in `terraform.tfvars` for step 1 and I see that this is hardcoded in https://github.com/vmware/terraform-provider-vcd/pull/1063/files#diff-11c94669df22bf8bb3ccf54e20fd388f624473c1f88a6847ec81d764f54d9ef6R31

Reply:
Yes, removed!